src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -964,6 +973,8 @@ namespace UiaOperationAbstraction
         }
     };
 
+#include "UiaTypeAbstractionEnums.g.h"

```

Comment:
This is probably fine for this PR, but I think we need to restructure UiaOperationAbstraction in the future. I briefly experimented with having two sets of files: one with only basic types (i.e. types which don't depend on other non-basic abstraction types) and one of non-basic types. Each of these sets would have generated files and un-generated files. I didn't fully test this so I'm not sure if there would be other compilation errors related to needing to keep generated and un-generated code separate. But the current structure makes it difficult to make changes without interleaving header/cpp files, so I feel like a refactoring would have a lot of benefits.

I suppose this also ties in with the current discussion about the code generator.